### PR TITLE
Add UI for PageSize

### PR DESCRIPTION
**Pull request by siyamandayubi**

### CODE REVIEW

**Comment by agriffard** [comment]
LGTM

**Comment by hishamco** [comment]
@siyamandayubi please resolve the conflict to do final review

File: src/OrchardCore.Modules/OrchardCore.Settings/Views/Settings.Edit.cshtml [hunk 1]
```diff
@@ -16,6 +16,15 @@
     <span class="hint">@T["Enter the fully qualified base URL of the web site."]</span>
 </fieldset>
 
+<div class="row">
+    <fieldset class="col-xl-6 form-group" asp-validation-class-for="PageSize">
+        <label asp-for="PageSize">@T["Page Size"]</label>
+        <input asp-for="PageSize" type="number" class="form-control" />
+        <span asp-validation-for="PageSize"></span>
+        <span class="hint">@T["The Page size that will be used by the Pagers in the Admin Panel of the web site."]</span>
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Settings/Views/Settings.Edit.cshtml]: The default page size. FIX maj typos. If you want to emphasize use bold of italic.

**Comment by agriffard** [comment]
@Skrypt Can you please resolve the conflicts?

**Comment by agriffard** [comment]
@siyamandayubi does not seem to answer. 
We should resolve the conflicts and merge it.

**Comment by deanmarcussen** [comment]
Agreed. I would use this immediately.

I'll resolve conflicts this morning

**Comment by scleaver** [comment]
Hooray!

**Comment by deanmarcussen** [comment]
Merged dev, and adjusted text as per @Skrypt request

oc-page-size-settings

And now I can see more tenants (and other things)on one page if I want too!

oc-tenants-bigger-list

**Comment by siyamandayubi** [comment]
Sorry guys for no response, I was in vacation and I didn't check my emails.

**Comment by Skrypt** [comment]
LGTM

**Comment by agriffard** [comment]
LGTM too :shipit:

**Pull request merged**